spec/acceptance/nodesets/default.yml

```diff
@@ -16,13 +16,14 @@ HOSTS:
 
   el8:
     platform:   el-8-x86_64
-    box:        generic/centos8
+    box:        centos/stream8
```

Comment:
Just FYI: https://app.vagrantup.com/generic/boxes/centos8 is updating regularly, but https://app.vagrantup.com/centos/boxes/stream8 hasn't been updated since Feb 2021.

Reply:
I think I had an issue with `generic/centos8` for some reason and switched over.

Reply:
Whoops, I meant https://app.vagrantup.com/generic/boxes/centos8s